docs/topics/getting-started-launchpad-create-mission.adoc

```diff
@@ -7,7 +7,10 @@ To create a mission:
 
 . Navigate to the Launchpad using your browser.
 . Select _Begin by selecting a functional booster_ and click _Next_.
-. Choose the booster you would like to run from the dropdown. Each booster illustrates a concept and has a specific runtime configured. You can come back to the Launchpad later and try another booster. 
+. Choose the mission and runtime you would like to run from the dropdowns, for example `rest-http` and `{VertX}`. 
```

Comment:
We'll probably no longer display the missions as a dropdown in the future, so this will change in the near future